Simplified/SimplyE.entitlements

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>com.apple.developer.associated-domains</key>
+	<array>
+		<string>applinks:skyneck.pl</string>
```

Comment:
I guess this was just for development? or do we need a NYPL url here?

Reply:
I removed the capability